packages/vscode-extension/src/webview/components/DeviceSelect.tsx

```diff
@@ -90,20 +89,21 @@ function DeviceSelect() {
   const stopPreviousDevices = use$(store$.workspaceConfiguration.stopPreviousDevices);
 
   const { selectedDeviceSession, projectState, project } = useProject();
-  const { devices, deviceSessionsManager } = useDevices();
+
+  const devices = use$(store$.devicesState.devices);
```

Comment:
```suggestion
  const devices = use$(store$.devicesState.devices) ?? [];
```
saves you on null checks elsewhere.